qiskit_metal/renderers/renderer_ansys/ansys_renderer.py

```diff
@@ -1029,6 +1043,63 @@ def add_mesh(self):
                 self.assign_mesh,
                 MaxLength=self._options['max_mesh_length_jj'])
 
+    #Still implementing
+    def auto_wirebonds(self, table):
+        """
+        Adds wirebonds to the Ansys model for path elements where;
+        subtract = True and wire_bonds = True.
+        Uses render options for determining of the;
+        * wb_threshold -- the minimum distance between two vertices of a path for a
+            wirebond to be added.
+        * wb_offset -- offset distance for wirebond placement (along the direction
+            of the cpw)
+        * wb_size -- controls the width of the wirebond (wb_size * path['width'])
+        """
+        norm_z = np.array([0, 0, 1])
+
+        wb_threshold = parse_units(self._options['wb_threshold'])
+        wb_offset = parse_units(self._options['wb_offset'])
+
+        #selecting only the qgeometry which meet criteria
+        wb_table = table.loc[table['hfss_wire_bonds'] == True]
+        wb_table2 = wb_table.loc[wb_table['subtract'] == True]
+
+        #looping through each qgeometry
+        for index, row in wb_table2.iterrows():
+            geom = row['geometry']
+            width = row['width']
+            #looping through the linestring of the path to determine where WBs should be
+            for index, i_p in enumerate(geom.coords[:-1], start=0):
```

Comment:
There are 2 nested 'for' loops here, each iterating over 'index'. The inner 'for' loop actually uses 'index'. To avoid confusion, perhaps rewrite line 1068 as `for _, row in wb_table2.iterrows()`?

Reply:
Good catch